google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -104,6 +122,12 @@ private SpannerOptions(Builder builder) {
     private int prefetchChunks = DEFAULT_PREFETCH_CHUNKS;
     private SessionPoolOptions sessionPoolOptions;
     private ImmutableMap<String, String> sessionLabels;
+    private SpannerStubSettings.Builder spannerStubSettingsBuilder =
```

Comment:
Can we remove the initialization here, as they are getting set to the options' builders in the below private constructor?

Reply:
No, it is needed, because the no-arg constructor on line 132 is called by `SpannerOptions.newBuilder()`, and does not contain any initialization code. The other fields that require initialization also do the initialization at  declaration instead of in the no-arg constructor. One option could be to move all these initializations from the declarations to the no-arg constructor, but my initial feeling is that it wouldn't make it clearer.